Payments/UserInterface/07a_ListLogging.md

```diff
@@ -0,0 +1,236 @@
+
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+    Click this button to close the *Detail of log message "Log message title"* view and return to the list of messages.    
+- [CANCEL]  
+    Click this button to close the *Detail of log message "Log message title"* view.
+
+- *Message*    
+    Description of the log message.
+
+
+
+### Detail of log message &ndash; Attributes
+
+*Payments > Logging > Tab CONNECTIONS > Select an intent > Select a log message > Tab Attributes*
+
+![Detail log message &ndash; Attributes](../../Assets/Screenshots/Payments/Logging/DetailLogMessageAttributes.png "[Detail Log Message &ndash; Attributes]")
+
+The *Attributes* tab displays further details of the selected log message. Depending on the attribute set of the log message, the displayed fields vary. All fields are read-only.
+
+In general, the most common attribute sets are the standard error log message attribute set as well as the API communication attribute set. The fields of these sets are described in the following. Furthermore, it is possible to create a custom log message attribute set with different fields.
+
+**Standard error log message attribute set**
+
+- *Error code*  
+    Number of the error code.
+
+- *File*  
+    Path of the file where the error has occurred.
+
+- *Line*  
+    Line in the file where the error has occurred.
+
+- *Exception class*   
+    Class name of the error. The error class describes the type of error that has occurred.
+
+- *Trace as string*   
+    Stack trace until the error has occurred. The stack trace lists the code until the error has occurred and is used to track the error.
+
+- *Exception*   
+    Detailed description of the error.
+
+**API communication attribute set**
+
+- *Code*   
+    Http response status code. For detailed information about the http status codes, see [http status codes](https://www.rfc-editor.org/rfc/rfc9110#section-15) website.
+
+- *Endpoint and method*   
+    Http method of the API request and url endpoint to which the request was sent.
+
+- *Request*   
+    Http header and body of the request.
+
+- *Response*   
+    Http header and body of the response.
+
+- *Exception*   
+    Description of the error. An exception is only displayed if an error has occurred and a http status code of the **4xx** class, for instance a **404** code has been returned.
+
+
+
+### Detail of log message &ndash; Dependencies   
+
+*Payments > Logging > Tab CONNECTIONS > Select an intent > Select a log message > Tab Dependencies*   
+
+![Detail of log message &ndash; Dependencies](../../Assets/Screenshots/Payments/Logging/DetailLogMessageDependenciespng.png "[Detailed of log message &ndash; Dependencies]")
+
+The *Payments* module only handles the communication between the payment service provider and Actindo. For this reason there are in the standard no dependencies to other Actindo entities that could be displayed here.
```

Comment:
"For this reason there are **in the standard** no dependencies to other Actindo entities that could be displayed here"
Ist der Satz vollständig? Meinst du as a standard, by default, oder in the standard module/UI?

Reply:
In the standard ist das Gleiche wie "by default"-> done